integration/spark/src/main/scala/org/apache/spark/sql/execution/command/view/CarbonDropMVCommand.scala

```diff
@@ -38,7 +38,8 @@ case class CarbonDropMVCommand(
     databaseNameOption: Option[String],
     name: String,
     ifExistsSet: Boolean,
-    forceDrop: Boolean = false)
+    forceDrop: Boolean = false,
+    lockAcquiredOnFactTable: String = null)
```

Comment:
```suggestion
    isLockAcquiredOnFactTable: String = null)
```

please change other places also